mlflow/pyfunc/__init__.py

```diff
@@ -397,6 +402,15 @@ def _enforce_mlflow_datatype(name, values: pandas.Series, t: DataType):
             raise MlflowException(
                 "Failed to convert column {0} from type {1} to {2}.".format(name, values.dtype, t)
             )
+    if t == DataType.double and values.dtype == decimal.Decimal:
+        # NB: Pyspark Decimal columne get converted to decimal.Decimal when converted to pandas
```

Comment:
nit: columne -> column.
in order to support ...  - is part of the comment missing?

Reply:
Thank you. Done.